packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -104,7 +159,34 @@ export const TranslationSidebar = ({
                 </div>
               )}
             </Tab.Panel>
-            <Tab.Panel>{t('common:coming_soon')}</Tab.Panel>
+            <Tab.Panel>
+              <div className="flex flex-col gap-2 pb-2">
+                {translatorNote?.authorName && (
```

Comment:
Let's add bold text here with the text `Translator's Notes`. We'll need that to distinguish between this input and the Footnote in #311.